src/components/common/Options/ContextOptions.tsx

```diff
@@ -0,0 +1,34 @@
+import { ContextOptionsProps } from './ContextOptions.types';
+
+export default function ContextOptions({
+  options = [
+    { id: 1, name: '적용하기' },
+    { id: 2, name: '수정하기' },
+    { id: 3, name: '삭제하기' },
+  ],
+  onClick,
+}: ContextOptionsProps) {
+  const firstOption = 'rounded-t-lg';
+  const bar = 'border-b border-d400 mx-2';
+  const lastOption = 'rounded-b-lg';
+  return (
```

Comment:
return문 전에는 개행을 넣어주는 습관을 들여주세요~
```suggestion
  const lastOption = 'rounded-b-lg';
  
  return (
```